cartographer/mapping/3d/submap_3d.cc

```diff
@@ -312,13 +310,11 @@ void ActiveSubmaps3D::InsertRangeData(
 void ActiveSubmaps3D::AddSubmap(const transform::Rigid3d& local_submap_pose) {
   if (submaps_.size() > 1) {
     submaps_.front()->Finish();
-    ++matching_submap_index_;
     submaps_.erase(submaps_.begin());
   }
   submaps_.emplace_back(new Submap3D(options_.high_resolution(),
                                      options_.low_resolution(),
                                      local_submap_pose));
-  LOG(INFO) << "Added submap " << matching_submap_index_ + submaps_.size();
```

Comment:
Can we retain a similar `LOG(INFO)`? Obviously it would have to be at a different place where we know the submap ID of the submap that got added.